doc/authentication.md

```diff
@@ -75,6 +76,13 @@ To allow establishing a new trust relationship from the client side, you must se
 In a production setup, unset `core.trust_password` after all clients have been added.
 This prevents brute-force attacks trying to guess the password.
 
+(authentication-token)=
+#### Adding client certificates using tokens
+
+New clients can be added using tokens. This is a safer way than using the trust password, as tokens expiry once they've been used. For this, the server needs to generate a token and pass it to the client.
+
+The server needs to call `lxc config trust add` which will prompt for a client name, and return a token. The client then calls `lxc remote add <name> <token>` to add the remote.
```

Comment:
It also works if you just run `lxc remote add <name>`. It will prompt for the trust password which also accepts a join token.

Reply:
Great! That fits what we documented before, so no bigger changes needed. :)

We should clarify this above though - something like:
```
   - If the client certificate is not in the server's trust store, the server checks if a trust password is set or if there is a token for the client. If there is, the server prompts the user for it.
     If the provided token or trust password matches, the client certificate is added to the server's trust store and the connection is granted.
     Otherwise, the connection is rejected.
   - If the client certificate is not in the server's trust store and no trust password or token is set, the connection is rejected.
```
Actually, is that correct? Will the server prompt only if a trust password or token is set, or will it always prompt and reject if whatever is provided isn't correct?

Reply:
```suggestion
```
This line is covered in the suggestion for line 82, so this one should be removed.
Just let me know if you'd prefer me to add a commit with the doc changes instead of commenting. ;)

Reply:
> Actually, is that correct? Will the server prompt only if a trust password or token is set, or will it always prompt and reject if whatever is provided isn't correct?

It will always prompt. The client doesn't know if a password is set or if there's a valid pending token.

> Just let me know if you'd prefer me to add a commit with the doc changes instead of commenting. ;)

Commenting is perfectly fine ;)

Reply:
OK, great, that makes the explanation a lot easier.
So lines 49-53 should be replaced with:
``` 
   - If the client certificate is in the server's trust store, the connection is granted.
   - If the client certificate is not in the server's trust store, the server prompts the user for a token or the trust password.
     If the provided token or trust password matches, the client certificate is added to the server's trust store and the connection is granted.
     Otherwise, the connection is rejected.